src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -1876,16 +1876,16 @@ protected String extractAuthorName(Document doc) {
             }
 
             if (authorName.isEmpty()) {  // for "schema.org creativework"
-                authorName = SHelper.innerTrim(doc.select("[itemtype=http://schema.org/Person]meta[itemprop=author]").attr("content"));
+                authorName = SHelper.innerTrim(doc.select("[itemtype$=schema.org/Person] meta[itemprop=author], [itemtype$=schema.org/Person] meta[itemprop=name]").attr("content"));
                 if(DEBUG_AUTHOR_EXTRACTION && !authorName.isEmpty()) System.out.println("AUTHOR: for \"schema.org creativework\" [itemtype=http://schema.org/Person]meta[itemprop=author]");
             }
 
             if (authorName.isEmpty()) {  // for "schema.org creativework"
-                result = doc.select("[itemtype=http://schema.org/Person]span[itemprop=author]").first();
+                result = doc.select("[itemtype$=schema.org/Person]span[itemprop=author], [itemtype$=schema.org/Person]span[itemprop=name]").first();
```

Comment:
This a good point, we probably should add tests to measure performance (i.e processing a set of articles N times) to try to detect large hits in performance. In the other hand this may be outside the score of this specific issue and may be better to handle in a separate PR.